range : 300
fix_topic : /sc30_driver_node/fix
#geographic_map_topic : /geographic_map
coast_line_csv_filename : coastline.csv
world_frame : world
enable_publish_marker : true